projects/storefrontlib/src/lib/store-finder/components/store-finder-search-result/store-finder-search-result.component.ts

```diff
@@ -76,7 +76,9 @@ export class StoreFinderSearchResultComponent implements OnInit, OnDestroy {
     }
 
     searchQuery.useMyLocation =
+      queryParams.useMyLocation != null &&
```

Comment:
use !==

Reply:
It won't work, since (null !== undefined) is true.